packages/plot/src/root.jsx

```diff
@@ -155,13 +166,14 @@ export class Root extends React.Component {
               side="left"
               text={labels.left}
               disabledLabel={disabledLabels}
-              placeholder={labelsPlaceholders.left}
+              placeholder={isChart ? labelsPlaceholders?.labels : labelsPlaceholders.left}
```

Comment:
i suggest to revert the changes to : `placeholder={labelsPlaceholders.left}` and to keep labelsPlaceholders an object with this structure `{ left: '', right: '', top: '', bottom: '' }`

Reply:
sounds good for me.